packages/svelte/elements.d.ts

```diff
@@ -2026,6 +2026,10 @@ export interface SvelteHTMLElements {
 		[name: string]: any;
 	};
 	'svelte:head': { [name: string]: any };
+	'svelte:boundary': {
+		onerror?: (error: Error, reset: () => void) => void;
```

Comment:
What do you think about adding a `onreset` prop here as well? There are some scenarios where executing a side effect could be useful. 

I know users can reach in and wrap the `reset` method, but that feels like it introduces friction.

Reply:
We already provide stack traces on our errors during DEV, this PR actually extends this so the stack happens in more places too.

Reply:
Awesome, I didn't realize we had the component stack on `error.message`, seems like it was implemented earlier! That's great, any error logging sdk can parse that and use right away.

Right now this component stack is attached to an error with `define_property(error, 'message'...` in `handle_error`. Could we evaluate defining it as a separate field on the error? This would make it easier for error logging to just grab and parse the field instead of having to read through `error.message`. It would also be nice to pass the `component_stack` into `onerror` callback so it can be used in userland (users don't have to parse `error.message` to get access to it).